ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -2831,3 +2831,84 @@ enum pkcs11_rc add_missing_attribute_id(struct obj_attrs **pub_head,
 	else
 		return set_attribute(pub_head, PKCS11_CKA_ID, id2, id2_size);
 }
+
+enum pkcs11_rc check_attr_checksum(struct obj_attrs *head)
```

Comment:
Could you rename this function `set_check_value_attr()` since it not only check to value but also sets it?